src/Components/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
```

Comment:
Thank you for including the screenshot in the PR message! This page looks really nice and handles the slightly awkward login UX really well